### PR TITLE
Upgrade to 7.10.2 ES

### PR DESCRIPTION
**Pull request by nirmalc**

### CODE REVIEW

**Comment by rinatkhaziev** [comment]
Since this is unmerged, I took the liberty and bumped it to 7.10.2. Per [release notes](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/release-notes-7.10.2.html) for 7.10.2 there's an information disclosure flaw and using 7.10.x is not recommended, 7.10.2 seems to be the final of 7.10

File: README.md [hunk 1]
```diff
@@ -100,12 +101,12 @@ The plugin artifacts are published to Maven Central and Github. To install a pre
...
 
 ```
-./bin/elasticsearch-plugin install https://github.com/Automattic/elasticsearch-statsd-plugin/releases/download/7.9.3.0/elasticsearch-statsd-7.9.3.0.zip
+./bin/elasticsearch-plugin install https://github.com/Automattic/elasticsearch-statsd-plugin/releases/download/7.8.1.0/elasticsearch-statsd-7.8.1.0.zip
 ```
 
 From Maven Central:
 ```
-./bin/elasticsearch-plugin install http://repo1.maven.org/maven2/com/automattic/elasticsearch-statsd/7.9.3.0/elasticsearch-statsd-7.9.3.0.zip
+./bin/elasticsearch-plugin install http://repo1.maven.org/maven2/com/automattic/elasticsearch-statsd/7.8.1.0/elasticsearch-statsd-7.8.1.0.zip
```
Comment (gibrown) [inline README.md:109]: is there a reason the readme references 7.8.1 rather than 7.10.2?

**Review by gibrown — APPROVED** [review]
This all looks fine to me other than the readme notes

File: README.md (hunk 1 above)
Comment (rinatkhaziev) [reply README.md:109]: umm, no, not really, It was in the PR already,  I just bumped pom.xml. Updating shortly.

**Pull request merged**